recipes-kernel/linux/linux-nilrt_git.bb

```diff
@@ -8,5 +8,5 @@ require linux-nilrt.inc
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-SRCREV_xilinx-zynq = "4442d176ee60b472252436950058a85fce08a3a3"
-SRCREV_qemu-zynq7 = "4442d176ee60b472252436950058a85fce08a3a3"
+SRCREV_xilinx-zynq = "f081753cec6a40d22e92fe90cff3b66690e8172f"
```

Comment:
This hash doesn't exist in `github.com/ni/linux`. If you intend to get https://github.com/ni/linux/commit/e167d2ccda8e812cd7aa99fefc59269c854952ce, hash should be `e167d2ccda8e812cd7aa99fefc59269c854952ce`

Reply:
I just updated it... I noticed it changed when the other PR was merged